### DISCUSSION TITLE
Translate Julia Doc To Persian

### DISCUSSION THREAD

**Discussion by he0xA1** [JuliaLang/julia#45994] [General]
Hello.
I want to translate Julia's document into Farsi language.
I wanted to know if it is possible for the translation to be available on the original document?
Or otherwise, how can I make it available to those interested?

**Comment by inkydragon** [comment]
> if it is possible for the translation to be available on the original document?

No sure, maybe we can ask @fredrikekre.

----

The Julia Chinese community ([JuliaCN](https://github.com/JuliaCN?type=source)) chose to create a new repo for translation work.

- source: https://github.com/JuliaCN/JuliaZH.jl
- website: https://docs.juliacn.com/latest/

The actual translation work is done on [Transifex](https://www.transifex.com/), using scripts for sync.
However, sync of the translated documents with the julia upstream repo still needs to be done manually.
So this is just how we do it, and there is still a lot of room for improvement.

**Comment by ViralBShah** [comment]
There was an effort earlier: https://julia-docs.readthedocs.io/fa/v1.6.0/index.html
